src/coreclr/tools/StressLogAnalyzer/StressLogAnalyzer.cpp

```diff
@@ -51,16 +69,30 @@ int ParseCommandLine(wchar_t* s, wchar_t** argv, int maxArgc)
     return argc;
 }
 
-int wmain(int argc, wchar_t *argv[])
+int ProcessStressLog(void* baseAddress, int argc, char* argv[]);
+
+int main(int argc, char *argv[])
 {
-    if (argc < 2 || wcscmp(argv[1], L"-?") == 0)
+#ifdef HOST_UNIX
+    int exitCode = PAL_InitializeDLL();
```

Comment:
This should call `PAL_Initialize` instead, the `PAL_InitializeDLL` is meant for shared libraries.